internal/config/config.go

```diff
@@ -110,7 +110,7 @@ func registerFlags() {
 		"info",
 		`The desired verbosity level for logging messages from nginx-agent. 
 		Available options, in order of severity from highest to lowest, are: 
-		panic, fatal, error, info, debug, and trace.`,
+		panic, fatal, error, info, debug and trace.`,
```

Comment:
```suggestion
		panic, fatal, error, info, debug, and trace.`,
```
Fan or not, style guide says we use the oxford comma. 😅